analytics/criteo.js

```diff
@@ -0,0 +1,109 @@
+//
+// Criteo tracking for auctions product feed and artworks product feed.
+//
+
+window.criteo_q = window.criteo_q || []
+var pathSplit = location.pathname.split("/")
+var userEmail = function() {
+  return sd.CURRENT_USER ? [sd.CURRENT_USER.email] : [];
+}()
+if (pathSplit[1] === "auctions") {
+  // Auctions events
+  window.criteo_q.push(
+    { event: "setAccount", account: sd.CRITEO_AUCTIONS_ACCOUNT_NUMBER },
+    { event: "setSiteType", type: "m" },
```

Comment:
I'm changing the `{ event: "setSiteType", ...}` of Auctions events to "m" from "d", as this is mobile Artsy.